chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/ChplRepeatableTriggerChangeEmailer.java

```diff
@@ -0,0 +1,130 @@
+package gov.healthit.chpl.scheduler;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import javax.mail.MessagingException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+
+import gov.healthit.chpl.dao.CertificationBodyDAO;
+import gov.healthit.chpl.domain.schedule.ChplRepeatableTrigger;
+import gov.healthit.chpl.exception.EntityRetrievalException;
+import gov.healthit.chpl.manager.SchedulerManager;
+import gov.healthit.chpl.util.EmailBuilder;
+import gov.healthit.chpl.util.HtmlEmailTemplate;
+import lombok.extern.log4j.Log4j2;
+
+
+@Service
+@Log4j2
+public class ChplRepeatableTriggerChangeEmailer {
+
+    private Environment environment;
+    private CertificationBodyDAO certificationBodyDao;
+    private String emailSubject;
+    private String emailBody;
+    private String emailStyles;
+
+    @Autowired
+    public ChplRepeatableTriggerChangeEmailer(Environment environment,
+            CertificationBodyDAO certificationBodyDao, @Value("${job.change.subject}") String emailSubject,
+            @Value("${job.change.body}") String emailBody, @Value("${email_styles}") String emailStyles) {
```

Comment:
I like the value loading here. Is this new? Have I just missed it before?

Reply:
It's how we're "supposed" to do it.  It makes mocking objects easier, too.